internal/config/flags.go

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) F5, Inc.
+ *
+ * This source code is licensed under the Apache License, Version 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package config
+
+import (
+	"time"
+
+	flag "github.com/spf13/pflag"
+)
+
+// Registrable registers a pflag
+type Registrable interface {
+	register(*flag.FlagSet)
+}
```

Comment:
I know this is likely copied code from v2 but per Go's naming guide this should be called `Registerer`: https://go.dev/doc/effective_go#interface-names

Reply:
Maybe name it something more appropriate with the -er suffix then?

Reply:
Removing Registrable as its not needed anymore